### PR TITLE
LPS-201012 | add summary in new macros | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# LPS-201012 | add summary in new macros

Hi! :)
This pr is corresponding to this ticket: [LPS-201012](https://liferay.atlassian.net/browse/LPS-201012)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/macros/LexiconList.macro [hunk 1]
```diff
@@ -23,7 +23,7 @@ definition {
 		MenuItem.clickNoError(menuItem = ${item});
 	}
 
-	@summary = "Default summary"
+	@summary = "Go to page by clicking on the breadcrumb link in the entry"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/LexiconList.macro:26]: Go to location by clicking on the breadcrumb item in the entry

File: portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/breadcrumb/Breadcrumb.macro [hunk 2]
```diff
@@ -54,7 +54,7 @@ definition {
 		SelectFrame(value1 = "relative=top");
 	}
 
-	@summary = "Default summary"
+	@summary = "Go to entry by clicking on the name link"
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/sitenavigation/breadcrumb/Breadcrumb.macro:57]: Go to breadcrumb entry

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @rafaellajordao, could you updating according to comments? Thanks

**Comment by rafaellajordao** [comment]
> Hi @rafaellajordao, could you updating according to comments? Thanks

done!

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**